contrib/fuzz/containerd_import_fuzzer.go

```diff
@@ -51,6 +58,8 @@ func fuzzContext() (context.Context, context.CancelFunc) {
 }
 
 func FuzzContainerdImport(data []byte) int {
+	initDaemon.Do(func(){startDaemon()})
```

Comment:
You probably can initDaemon.Do(startDaemon)

Reply:
Yep, works great locally. Thanks!